src/options/views/script-item.vue

```diff
@@ -366,6 +375,8 @@ export default {
   border-radius: .3rem;
   transition: transform .5s;
   background: white;
+  /* image size + padding + margin + border + 1 */
+  height: calc(3rem + 28px);
```

Comment:
I'd like to use postcss compile-time constants/variables for these calculations.
Need help since I'm a noob at this.